skills/odl-pdf/SKILL.md

```
@@ -0,0 +1,740 @@
+name: odl-pdf
+description: >
+  Expert PDF extraction guidance for opendataloader-pdf. Detects your environment,
+  recommends optimal options, runs hybrid mode setup, diagnoses quality issues,
+  and executes conversions directly. Use when: 'PDF extraction', 'PDF to markdown',
+  'PDF to JSON', 'PDF to HTML', 'opendataloader', 'ODL', 'hybrid mode',
+  'scanned PDF', 'OCR', 'PDF tables', 'RAG pipeline with PDF', 'PDF accessibility',
+  'PDF/UA'. Do NOT use for: PDF merge/split/rotate, Word/Excel conversion,
+  PDF form filling.
+---
+
+# Targets: opendataloader-pdf >= 2.2.0
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)